modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -26,7 +26,8 @@ Follow these steps to use OpenSSL:
 Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
 Which dependencies you need depends upon the Neo4j version.
 Each version of Neo4j ships with a version of Netty and Netty requires specific tcnative versions.
-See table below.
+Make sure to install the version that matches your OS processor.
+For more details, see the <<table, table>>.
```

Comment:
You are so sweet. I meant something like this, but feel free to change it if you'd like:
```suggestion
For more details, see the <<table, Netty support per Neo4j version>>.
```

Reply:
ah ok 😅 thanks for the clarification!!